cpp/velox/substrait/SubstraitParser.cc

```diff
@@ -317,11 +316,13 @@ std::string SubstraitParser::findVeloxFunction(
 std::string SubstraitParser::mapToVeloxFunction(const std::string& substraitFunction, bool isDecimal) {
   auto it = substraitVeloxFunctionMap_.find(substraitFunction);
   if (isDecimal) {
-    if (substraitFunction == "add" || substraitFunction == "subtract" || substraitFunction == "multiply" ||
-        substraitFunction == "divide" || substraitFunction == "avg" || substraitFunction == "avg_merge" ||
-        substraitFunction == "sum" || substraitFunction == "sum_merge" || substraitFunction == "round") {
+    if (substraitFunction == "round" || substraitFunction == "lt" || substraitFunction == "lte" ||
```

Comment:
Does it mean that functions like "devide" won't be supported after switching to upstream velox?

Reply:
No, it's just a name mapping. For decimal type, decimal_divide will used.